### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by danielcber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Get Github Shortname _(get-github-shortname)_
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
```
Comment (RichardLitt) [inline README.md]: I've been leaning away from using this badge recently. It's fine for this repo (it's mine, after all), but in the future, this addition is probably unnecessary.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+> Building Documentation Best Practices
```
Comment (RichardLitt) [inline README.md]: This should match the description for the repository, which you can see above the README when looking at the repository. The description for this repository should be: "Get the GitHub repository name"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Great documentation takes work. It should make it easy for newcomers to contribute to projects and understand your project! Here you will find the instructions for how to make your documentation as accessible as possible.
```
Comment (RichardLitt) [inline README.md]: This doesn't describe this repository.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+## Table of Contents
+* [Contribute](https://github.com/RichardLitt/get-github-shortname#contribute)
+* [License](https://github.com/RichardLitt/get-github-shortname#license)
+
+
```
Comment (RichardLitt) [inline README.md]: Needs install and usage sections. Putting `TODO` in them is fine, for now.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+## Contribute
+Feel free to dive in! Open an [issue or submit PRs](https://github.com/RichardLitt/get-github-shortname/issues/new).
+
+Standard Readme follows the [Contributor Covenant Code of Conduct](http://contributor-covenant.org/version/1/3/0/).
```
Comment (RichardLitt) [inline README.md]: This isn't Standard Readme. Change to the name of this repository.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+## License
+UNLICENSED
```
Comment (RichardLitt) [inline README.md]: The package.json suggests `ISC`. These should match. Change this to MIT, and change that to MIT, too.

File: README.md (hunk 1 above)
Comment (danielcber) [reply README.md]: Noted!

Comment (danielcber) [reply README.md]: How about I take it off then, its better to not establish the practice from the get go.

File: README.md (hunk 2 above)
Comment (danielcber) [reply README.md]: Got it!

File: README.md (hunk 4 above)
Comment (danielcber) [reply README.md]: Done.

File: README.md (hunk 5 above)
Comment (danielcber) [reply README.md]: Done.

File: README.md (hunk 6 above)
Comment (danielcber) [reply README.md]: Done.

**Comment by RichardLitt** [comment]
Sweet. Note: This closes #4.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+# Get Github Shortname _(get-github-shortname)_
+
+> Get the GitHub repository name
+
+A shortcut to getting a Github repository's name using [git-remote-upstream-url](https://www.npmjs.com/package/git-remote-upstream-url) and [git-remote-origin-url](https://www.npmjs.com/package/git-remote-origin-url).
+
+## Table of Contents
+* [Install](https://github.com/RichardLitt/get-github-shortname#install)
```
Comment (RichardLitt) [inline README.md]: These can just be `#install`, instead of the entire URL. Want to change them? :)

File: README.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+TODO
+
+## Contribute
+Feel free to dive in! Open an [issue](https://github.com/RichardLitt/get-github-shortname/issues/new) or submit PRs.
+
+Get Github Shortname follows the [Contributor Covenant Code of Conduct](http://contributor-covenant.org/version/1/3/0/).
+
+## License
+
+The MIT License (MIT)
```
Comment (RichardLitt) [inline README.md]: Huh. Only the short version is needed here: "[MIT](LICENSE.md) © 2018 Richard Littauer". And then, we need another license file with the rest of this content in it. That file should be called `LICENSE.md`.

File: README.md [hunk 9]
```diff
@@ -22,25 +22,4 @@ Feel free to dive in! Open an [issue](https://github.com/RichardLitt/get-github-
...
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+[MIT](https://github.com/RichardLitt/get-github-shortname/blob/master/LICENSE.md) © 2018 Richard Littauer
```
Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

File: README.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+## Usage
+TODO
+
+## Contribute
+Feel free to dive in! Open an [issue](https://github.com/RichardLitt/get-github-shortname/issues/new) or submit PRs.
+
+Get Github Shortname follows the [Contributor Covenant Code of Conduct](http://contributor-covenant.org/version/1/3/0/).
+
+## License
+[MIT](https://github.com/RichardLitt/get-github-shortname/blob/master/LICENSE.md) © 2018 Richard Littauer
```
Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

Comment (RichardLitt) [inline README.md]: Just link like this:

`[MIT](LICENSE.md) © 2018 Richard Littauer` 

:)

**Comment by RichardLitt** [comment]
Looks good!

**Pull request merged**